packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/useVotes.ts

```diff
@@ -49,12 +38,11 @@ async function Suspender(id: ProposalIdentifier['id'], space: ProposalIdentifier
                     : undefined,
                 address: v.voter,
                 authorIpfsHash: v.ipfs ?? v.id,
-                balance: sumBy(scores, (score) => score[v.voter.toLowerCase()] ?? 0),
-                scores: strategies.map((_strategy, i) => scores[i][v.voter] || 0),
+                balance: v.vp,
+                scores: [],
```

Comment:
Here `v.vp` can be assigned to `balance`, and `scores` is useless.

But please check
`packages/mask/src/plugins/Snapshot/SNSAdaptor/VotingCard.tsx`, 
`packages/mask/src/plugins/Snapshot/SNSAdaptor/hooks/usePower.ts`,
The voting power is derived from the `scores`, if you can find something to generate the voting power, then `PluginSnapshotRPC.getScores` can be removed.